pkg/instancegroups/instancegroups.go

```diff
@@ -90,7 +90,7 @@ func (c *RollingUpdateCluster) rollingUpdateInstanceGroup(cluster *api.Cluster,
 	} else if c.CloudOnly {
 		klog.V(3).Info("Not validating cluster as validation is turned off via the cloud-only flag.")
 	} else {
-		if err = c.validateCluster(); err != nil {
+		if err = c.validateClusterWithDuration(validationTimeout); err != nil {
```

Comment:
In my opinion this is good change, I was planning to make same.

Reply:
```
pkg/instancegroups/instancegroups.go:458:32: func (*RollingUpdateCluster).validateCluster is unused (U1000)
```

that is needed to remove if we continue here. I can add lgtm, but that fix is needed first